CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 if (WIN32)
 cmake_minimum_required(VERSION 3.20 FATAL_ERROR)
 else()
-cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
```

Comment:
Could you explain why you moved it to `3.5`?

Reply:
I wouldn't mind moving it up higher even: 1) `liboqs` [demands 3.15](https://github.com/open-quantum-safe/liboqs/blob/2acfd964b95affd355da080ce9ea9bddd776b708/CMakeLists.txt#L3) and 2) the fewer versions permitted the less support hassle for us.

Reply:
ho I didn't know for `liboqs`, thank you @baentsch and sorry for the noise here…

Reply:
The warning i got indicated < 3.5 will be deprecated ....
"CMake Deprecation Warning at CMakeLists.txt:4 (cmake_minimum_required…
…): Compatibility with CMake < 3.5 will be removed from a future version of CMake."